### PR TITLE
[ENT-451] Fix formatting nits on Enterprise landing page

### PR DESCRIPTION
**Pull request by haikuginger**
**Description:**
This pull request fixes two nits:

1. The course final price (in dollars) should have two decimal places if not an integer. If an integer, it should have no decimal places.
2. If one hour of effort per week is required, a singular noun should be used to describe that single hour.

We fix point 1 using a quick integer check and string formatting operation. We fix point 2 by using `ungettext` instead of `ugettext` for this particular string.

**JIRA:** [ENT-451](https://openedx.atlassian.net/browse/ENT-451)

**Screenshot:**
screen shot 2017-07-06 at 1 46 55 pm

**Dependencies:** None 

**Merge deadline:** 16 July 2017

**Installation instructions:**
Requires course discovery/ecommerce
Requires configured Enterprise entitlement for course with discount
Requires course to be configured in Studio with one hour of effort per week.

**Testing instructions:**

1. Browse to the Enterprise course enrollment page (`http://localhost:8000/enterprise/176596e9-b93b-4a15-97eb-65d26ea309ea/course/course-v1%3AedX%2BDemoX%2BDemo_Course/enroll/`).
2. Click the View Course Details link.
3. Verify that the final price is presented without a decimal place if the discount results in an integer price, or with two decimal places if the discount results in any other price.
4. Verify that the Effort is described as "1 hour per week, per course".

**Merge checklist:**

- [x] Regenerate requirements with `make upgrade && make requirements` (and make sure to fix any errors).
  **DO NOT** just add dependencies to `requirements/*.txt` files.
- [x] All reviewers approved
- [x] CI build is green
- [x] Version bumped
- [x] Changelog record added
- [x] Documentation updated (not only docstrings)
- [x] Commits are (reasonably) squashed
- [x] Translations are updated
- [x] PR author is listed in AUTHORS

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPi after tag-triggered build is finished.
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @haikuginger! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fedx-enterprise&number=145"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

**Comment by haikuginger** [comment]
@douglashall, can we get this on the Business sandbox?

File: enterprise/views.py [hunk 1]
```diff
@@ -758,9 +759,12 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
...
             course_effort = ''
         else:
-            course_effort = self.context_data['effort_hours_text'].format(
-                hours=effort_hours
-            )
+            course_effort = ungettext(
+                '{hours} hour per week, per course',
+                '{hours} hours per week, per course',
+                effort_hours,
+            ).format(hours=effort_hours)
```
Comment (UmanShahzad) [inline enterprise/views.py]: Using `ungettext` was a smart move!

Nit: Do you think it'd be possible, though, to stick this function unformatted into `context_data` under the same key we used before? We can then re-use it here (and maybe elsewhere) as it was being used previously exactly the same way.

Comment (haikuginger) [inline enterprise/views.py]: @UmanShahzad, we could potentially do it as a lambda on `context_data`. The trick is that we need the numeric variable in question to do the `ungettext` call.

**Comment by UmanShahzad** [comment]
@haikuginger The code changes / tests look good to me! I'm going to manually test it next.

File: enterprise/views.py (hunk 1 above)
Comment (UmanShahzad) [inline enterprise/views.py]: Meh, if it makes the code look even more complicated then nevermind.

**Comment by chris-mike** [comment]
A few more nits: Any chance we can reduce the h-space here and here (see big yellow dots)? I thought previous versions had different/smaller use of space.
screen shot 2017-07-06 at 4 18 47 pm

**Comment by UmanShahzad** [comment]
@chris-mike Yep, but probably better for a different PR.

**Comment by chris-mike** [comment]
I moved my comment to https://openedx.atlassian.net/browse/ENT-451, which seems to be active.

**Comment by haikuginger** [comment]
@chris-mike, I've updated the code to reduce the padding on those elements:

screen shot 2017-07-10 at 8 45 28 am

**Comment by chris-mike** [comment]
👍 Looks good. Thanks

**Pull request merged**